### PR TITLE
make it works on rails 5.1

### PR DESCRIPTION
**Pull request by aronwolf90**
it fix the bug in rails 5.1 (In Rails 5.1 secondbase does not work).

### CODE REVIEW

**Comment by aronwolf90** [comment]
maybe I should make a rebase on https://github.com/customink/secondbase/pull/45 to include the tests?

**Comment by metaskills** [comment]
> maybe I should make a rebase on #45 to include the tests?

Yea, or duplicate that work here. We wont know if it really works on 5.1 till all our standard tests pass.

File: test/dummy_apps/rails_five/Rakefile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-require File.expand_path '../init', __FILE__
+require_relative File.expand_path '../config/init', __FILE__
```
Comment (metaskills) [inline test/dummy_apps/rails_five/Rakefile]: Nice cleanup! Can this be shortened to `require_relative '../config/init'`

**Comment by metaskills** [comment]
Published v2.1.4 with Rails v5.1 fixes. Thanks!

**Pull request merged**